docs/source/optim.rst

```diff
@@ -270,16 +270,44 @@ SWA has been proposed in `Averaging Weights Leads to Wider Optima and Better Gen
 Constructing averaged models
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-`AveragedModel` class serves to compute the weights of the SWA model. You can create an
-averaged model by running:
+The `AveragedModel` class serves to compute the weights of the SWA or EMA model.
+
+You can create an SWA averaged model by running:
+
+>>> averaged_model = AveragedModel(model)
 
->>> swa_model = AveragedModel(model)
+EMA models are constructed by specifying the ``multi_avg_fn`` argument as follows:
 
-Here the model ``model`` can be an arbitrary :class:`torch.nn.Module` object. ``swa_model``
+>>> averaged_model = AveragedModel(model, multi_avg_fn=get_ema_multi_avg_fn(decay))
```

Comment:
Should we also note that decay is something the user has to provide? Better yet, could it be made optional where if it's not provided, decay will default to a certain value?

Reply:
Added default value and formula